tests/queries/0_stateless/02995_index_7.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-# Tags: long, no-debug, no-asan, no-tsan, no-msan, no-ubsan, no-sanitize-coverage
+# Tags: long, no-debug, no-asan, no-tsan, no-msan, no-ubsan, no-sanitize-coverage, no-distributed-cache
```

Comment:
Probably all `02995_index_*` could be updated in a such way

Reply:
I think all other should already be updated this way, I remember I was adding them, but probably missed this one